src/NotepadNext/docks/FolderAsWorkspaceDock.cpp

```diff
@@ -18,20 +18,27 @@
 
 
 #include "FolderAsWorkspaceDock.h"
+
+#include <QFileSystemModel>
+#include <QMessageBox>
+
 #include "ApplicationSettings.h"
+#include "MainWindow.h"
 #include "ui_FolderAsWorkspaceDock.h"
 
-#include <QFileSystemModel>
+QString NEW_DIR_TEMPLATE("dir_%1");
```

Comment:
I'd have to look deeper how this is used but might make sense to keep it inside the class so this can be translated.

Reply:
done